code/game/objects/effects/overlays.dm

```diff
@@ -132,6 +132,28 @@
 	duration = 8
 	randomdir = 0
 
+/obj/effect/overlay/temp/gib_animation
+	icon = 'icons/mob/mob.dmi'
+	duration = 15
```

Comment:
With some gib animations this could potentially loop, but it'd be easy to fix that just by going into the sprite file and setting the gib/dust icon to loop only once.